samples/highcharts/demo/spline-irregular-time/demo.js

```diff
@@ -42,123 +44,281 @@ Highcharts.chart('container', {
-            [Date.UTC(1971, 0, 11), 0.89],
-            [Date.UTC(1971, 0, 17), 1.04],
-            [Date.UTC(1971, 0, 20), 1.02],
-            [Date.UTC(1971, 0, 25), 1.03],
-            [Date.UTC(1971, 0, 30), 1.39],
-            [Date.UTC(1971, 1,  5), 1.77],
-            [Date.UTC(1971, 1, 26), 2.12],
-            [Date.UTC(1971, 3, 19), 2.1],
-            [Date.UTC(1971, 4,  9), 1.7],
-            [Date.UTC(1971, 4, 29), 0.85],
-            [Date.UTC(1971, 5,  7), 0]
-        ]
-    }, {
-        name: "Winter 2016-2017",
-        data: [
-            [Date.UTC(1970, 9, 15), 0],
-            [Date.UTC(1970, 9, 31), 0.09],
-            [Date.UTC(1970, 10,  7), 0.17],
-            [Date.UTC(1970, 10, 10), 0.1],
-            [Date.UTC(1970, 11, 10), 0.1],
-            [Date.UTC(1970, 11, 13), 0.1],
-            [Date.UTC(1970, 11, 16), 0.11],
-            [Date.UTC(1970, 11, 19), 0.11],
-            [Date.UTC(1970, 11, 22), 0.08],
-            [Date.UTC(1970, 11, 25), 0.23],
-            [Date.UTC(1970, 11, 28), 0.37],
-            [Date.UTC(1971, 0, 16), 0.68],
-            [Date.UTC(1971, 0, 19), 0.55],
-            [Date.UTC(1971, 0, 22), 0.4],
-            [Date.UTC(1971, 0, 25), 0.4],
-            [Date.UTC(1971, 0, 28), 0.37],
-            [Date.UTC(1971, 0, 31), 0.43],
-            [Date.UTC(1971, 1,  4), 0.42],
-            [Date.UTC(1971, 1,  7), 0.39],
-            [Date.UTC(1971, 1, 10), 0.39],
-            [Date.UTC(1971, 1, 13), 0.39],
-            [Date.UTC(1971, 1, 16), 0.39],
-            [Date.UTC(1971, 1, 19), 0.35],
-            [Date.UTC(1971, 1, 22), 0.45],
-            [Date.UTC(1971, 1, 25), 0.62],
-            [Date.UTC(1971, 1, 28), 0.68],
-            [Date.UTC(1971, 2,  4), 0.68],
-            [Date.UTC(1971, 2,  7), 0.65],
-            [Date.UTC(1971, 2, 10), 0.65],
-            [Date.UTC(1971, 2, 13), 0.75],
-            [Date.UTC(1971, 2, 16), 0.86],
-            [Date.UTC(1971, 2, 19), 1.14],
-            [Date.UTC(1971, 2, 22), 1.2],
-            [Date.UTC(1971, 2, 25), 1.27],
-            [Date.UTC(1971, 2, 27), 1.12],
-            [Date.UTC(1971, 2, 30), 0.98],
-            [Date.UTC(1971, 3,  3), 0.85],
-            [Date.UTC(1971, 3,  6), 1.04],
-            [Date.UTC(1971, 3,  9), 0.92],
-            [Date.UTC(1971, 3, 12), 0.96],
-            [Date.UTC(1971, 3, 15), 0.94],
-            [Date.UTC(1971, 3, 18), 0.99],
-            [Date.UTC(1971, 3, 21), 0.96],
-            [Date.UTC(1971, 3, 24), 1.15],
-            [Date.UTC(1971, 3, 27), 1.18],
-            [Date.UTC(1971, 3, 30), 1.12],
-            [Date.UTC(1971, 4,  3), 1.06],
-            [Date.UTC(1971, 4,  6), 0.96],
-            [Date.UTC(1971, 4,  9), 0.87],
-            [Date.UTC(1971, 4, 12), 0.88],
-            [Date.UTC(1971, 4, 15), 0.79],
-            [Date.UTC(1971, 4, 18), 0.54],
-            [Date.UTC(1971, 4, 21), 0.34],
-            [Date.UTC(1971, 4, 25), 0]
-        ]
-    }],
+    series: [
```

Comment:
This demo previously had 3 series, now 4 is proposed. But I think it looks a bit cluttered with four series - better to remove one of them?